### PR TITLE
DM-7036: Port to Python 3

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: tests/Dictionary_1.py [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
 import os
 import unittest
 import lsst.utils
-#import inspect
+# import inspect
```
Comment (PaulPrice) [inline tests/Dictionary_1.py]: Let's remove that line completely.

File: tests/Policy_1.py [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
-# You should have received a copy of the LSST License Statement and 
-# the GNU General Public License along with this program.  If not, 
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
-#import pdb                          # we may want to say pdb.set_trace()
+# import pdb                          # we may want to say pdb.set_trace()
```
Comment (PaulPrice) [inline tests/Policy_1.py]: Let's remove this line.  If anyone wants pdb, they can import it themselves.

File: tests/Policy_1.py [hunk 3]
```diff
@@ -112,14 +112,14 @@ def testPolicySetget(self):
...
+        # n = mwid.Citizen_census(0)
         p = Policy.createPolicy("examples/EventTransmitter_policy.paf")
         self.assertEquals(p.get("transmitter.serializationFormat"), "deluxe")
         p = None
-#        self.assertEquals(mwid.Citizen_census(0), n, "Policy apparently leaked")
+        # self.assertEquals(mwid.Citizen_census(0), n, "Policy apparently leaked")
 
     def testEmptyPolicy(self):
-        p = Policy()
+        p = Policy()  # noqa
```
Comment (PaulPrice) [inline tests/Policy_1.py]: Where'd that comment come from, and what does it mean?

File: tests/UrnPolicyFile_1.py [hunk 4]
```diff
@@ -1,42 +1,44 @@
...
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
 import os
 import unittest
 import lsst.utils
-#import inspect
+# import inspect
```
Comment (PaulPrice) [inline tests/UrnPolicyFile_1.py]: Remove.

File: tests/testBigBool.py [hunk 5]
```diff
@@ -1,58 +1,56 @@
...
 #
 
 """
 Comprehensive tests reading and retrieving data of all types
 """
 from __future__ import print_function
-import pdb                              # we may want to say pdb.set_trace()
-import os
-import sys
+# import pdb                              # we may want to say pdb.set_trace()
```
Comment (PaulPrice) [inline tests/testBigBool.py]: Kill.

File: tests/testGet.py [hunk 6]
```diff
@@ -1,45 +1,44 @@
...
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
 """
 Comprehensive tests reading and retrieving data of all types
 """
-import pdb                              # we may want to say pdb.set_trace()
+# import pdb                              # we may want to say pdb.set_trace()
```
Comment (PaulPrice) [inline tests/testGet.py]: Kill.

File: tests/testGet.py [hunk 7]
```diff
@@ -99,28 +98,28 @@ def testGetDoublArray(self):
...
         for i in xrange(len(truth)):
             self.assertAlmostEquals(v[i], truth[i], 8,
-                              "wrong array element at index %d: %g != %g" %
-                              (i, v[i], truth[i]))
+                                    "wrong array element at index %d: %g != %g" %
+                                    (i, v[i], truth[i]))
 
     def testGetStringArray(self):
-        #pdb.set_trace()
+        # pdb.set_trace()
```
Comment (PaulPrice) [inline tests/testGet.py]: Kill.

File: tests/testPolicyString.py [hunk 8]
```diff
@@ -1,38 +1,36 @@
...
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
 """
 Comprehensive tests reading and retrieving data of all types
 """
-import pdb                              # we may want to say pdb.set_trace()
-import os
-import sys
+# import pdb                              # we may want to say pdb.set_trace()
```
Comment (PaulPrice) [inline tests/testPolicyString.py]: Another.

File: tests/testStringDest.py [hunk 9]
```diff
@@ -1,38 +1,36 @@
...
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
 """
 Comprehensive tests reading and retrieving data of all types
 """
-import pdb                              # we may want to say pdb.set_trace()
-import os
-import sys
+# import pdb                              # we may want to say pdb.set_trace()
```
Comment (PaulPrice) [inline tests/testStringDest.py]: Another.

File: tests/Policy_1.py (hunk 3 above)
Comment (timj) [inline tests/Policy_1.py]: I've annotated it now so you know it means variable assigned but unused.

**Pull request merged**